src/lib/plugins/blocks.js

```diff
@@ -13,19 +13,19 @@ module.exports.player = function (player, serv) {
     else serv.updateBlock(player.world, position, serv.tickCount, serv.tickCount, true)
   }
 
-  player.sendBlock = (position, blockType, blockData) => // Call from player.setBlock unless you want "local" fake blocks
+  player.sendBlock = (position, blockType, blockData, blockStateId) => // Call from player.setBlock unless you want "local" fake blocks
```

Comment:
you can remove blockType and blockData, it's redundant with blockStateId

Reply:
done